test/MockAccessToken.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env mocha */
+
+// Technically, this doesn't really look like a 'mock' function thing, but
+//    hopefully you get the idea behind it.
+// 'require' this in tests that need to pass accessToken-based authentication.
+//    In the test's primary 'describe' block, enter
+//      require('../BeforeAndAfter.js')
+//      as the first line inside of the describe block
+//    Look at test/auth/AccessTokenExample.test.js for more infos
+
+const { TestRedisAdapter, testAccessToken, testUserId } = require('./testConfig')
+
+// get a Redis
+const adapter = new TestRedisAdapter()
+
+// Give a test access token to a Redis. Then Redis will believe there is a valid login.
+before('send access token to Redis', () => {
```

Comment:
just a thought for the future I think it would be useful to be able to be more flexible around which userIds get sent to redis for this, in case we need to have a user with different permissions, for example.